### PR TITLE
update stable version of fast foundation to provide bug fixes

### PR DESCRIPTION
**Pull request by chrisdholt**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/dffe6fbbe4e402b722feb8e2d5252656d2905fdd'>dffe6fbbe4e402b722feb8e2d5252656d2905fdd</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Avatar-mount_base.html">871</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Avatar-mount.html">892</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Button-mount_base.html">476</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Button-mount.html">476</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Field-mount_base.html">1557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Field-mount.html">1695</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProvider-mount_base.html">1150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProvider-mount.html">1161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProviderWithTheme-mount_base.html">152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProviderWithTheme-mount.html">148</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">122</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">120</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">134</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/InfoButton-mount_base.html">27</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/InfoButton-mount.html">30</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/MakeStyles-mount_base.html">1456</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/MakeStyles-mount.html">1394</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Persona-mount_base.html">2614</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/Persona-mount.html">2461</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/SpinButton-mount_base.html">1948</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-react-components/SpinButton-mount.html">1982</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27506&commitId=230f0f25e60f5196d0484af13ad035e791125ff3&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
AttachmentMinimalPerf.default | [132](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AttachmentMinimalPerf.default.html) | [119](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.11:1
VideoMinimalPerf.default | [644](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/VideoMinimalPerf.default.html) | [597](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.08:1
LoaderMinimalPerf.default | [294](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/LoaderMinimalPerf.default.html) | [276](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.07:1
RefMinimalPerf.default | [169](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/RefMinimalPerf.default.html) | [158](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.07:1
FlexMinimalPerf.default | [244](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/FlexMinimalPerf.default.html) | [231](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.06:1
FormMinimalPerf.default | [332](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/FormMinimalPerf.default.html) | [316](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.05:1
BoxMinimalPerf.default | [301](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/BoxMinimalPerf.default.html) | [289](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.04:1
ButtonOverridesMissPerf.default | [1004](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [968](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.04:1
TableMinimalPerf.default | [351](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TableMinimalPerf.default.html) | [338](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.04:1
DropdownManyItemsPerf.default | [574](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DropdownManyItemsPerf.default.html) | [556](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.03:1
HeaderSlotsPerf.default | [689](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/HeaderSlotsPerf.default.html) | [669](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.03:1
ListMinimalPerf.default | [454](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListMinimalPerf.default.html) | [442](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.03:1
ListNestedPerf.default | [474](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListNestedPerf.default.html) | [459](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListNestedPerf.default_base.html) | 1.03:1
PopupMinimalPerf.default | [541](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/PopupMinimalPerf.default.html) | [527](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.03:1
ProviderMergeThemesPerf.default | [1174](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1142](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.03:1
TextAreaMinimalPerf.default | [406](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TextAreaMinimalPerf.default.html) | [393](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.03:1
CarouselMinimalPerf.default | [398](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CarouselMinimalPerf.default.html) | [391](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.02:1
GridMinimalPerf.default | [292](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/GridMinimalPerf.default.html) | [285](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.02:1
HeaderMinimalPerf.default | [322](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/HeaderMinimalPerf.default.html) | [315](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.02:1
ItemLayoutMinimalPerf.default | [1080](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [1061](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.02:1
MenuMinimalPerf.default | [747](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/MenuMinimalPerf.default.html) | [729](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.02:1
TextMinimalPerf.default | [293](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TextMinimalPerf.default.html) | [286](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.02:1
AttachmentSlotsPerf.default | [993](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AttachmentSlotsPerf.default.html) | [982](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.01:1
ChatMinimalPerf.default | [609](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ChatMinimalPerf.default.html) | [602](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.01:1
DropdownMinimalPerf.default | [2138](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DropdownMinimalPerf.default.html) | [2117](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.01:1
InputMinimalPerf.default | [845](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/InputMinimalPerf.default.html) | [837](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.01:1
LabelMinimalPerf.default | [323](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/LabelMinimalPerf.default.html) | [320](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.01:1
PortalMinimalPerf.default | [141](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/PortalMinimalPerf.default.html) | [139](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.01:1
SliderMinimalPerf.default | [1057](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SliderMinimalPerf.default.html) | [1048](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.01:1
AccordionMinimalPerf.default | [114](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AccordionMinimalPerf.default.html) | [114](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1:1
AlertMinimalPerf.default | [227](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AlertMinimalPerf.default.html) | [226](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1:1
AnimationMinimalPerf.default | [420](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AnimationMinimalPerf.default.html) | [421](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1:1
DatepickerMinimalPerf.default | [5235](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DatepickerMinimalPerf.default.html) | [5224](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1:1
ImageMinimalPerf.default | [322](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ImageMinimalPerf.default.html) | [322](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1:1
LayoutMinimalPerf.default | [308](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/LayoutMinimalPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1:1
ListWith60ListItems.default | [551](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListWith60ListItems.default.html) | [549](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListWith60ListItems.default_base.html) | 1:1
SplitButtonMinimalPerf.default | [3532](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [3535](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1:1
StatusMinimalPerf.default | [597](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/StatusMinimalPerf.default.html) | [599](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1:1
TreeMinimalPerf.default | [686](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TreeMinimalPerf.default.html) | [684](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1:1
AvatarMinimalPerf.default | [157](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AvatarMinimalPerf.default.html) | [158](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 0.99:1
ButtonSlotsPerf.default | [476](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ButtonSlotsPerf.default.html) | [483](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 0.99:1
CardMinimalPerf.default | [435](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CardMinimalPerf.default.html) | [440](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.99:1
CheckboxMinimalPerf.default | [1844](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1867](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 0.99:1
RosterPerf.default | [2164](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/RosterPerf.default.html) | [2175](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/RosterPerf.default_base.html) | 0.99:1
ReactionMinimalPerf.default | [319](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ReactionMinimalPerf.default.html) | [323](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 0.99:1
SkeletonMinimalPerf.default | [294](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SkeletonMinimalPerf.default.html) | [297](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 0.99:1
IconMinimalPerf.default | [534](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/IconMinimalPerf.default.html) | [541](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/IconMinimalPerf.default_base.html) | 0.99:1
TableManyItemsPerf.default | [1682](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TableManyItemsPerf.default.html) | [1696](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TableManyItemsPerf.default_base.html) | 0.99:1
CustomToolbarPrototype.default | [2193](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CustomToolbarPrototype.default.html) | [2207](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 0.99:1
DialogMinimalPerf.default | [689](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DialogMinimalPerf.default.html) | [704](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DialogMinimalPerf.default_base.html) | 0.98:1
EmbedMinimalPerf.default | [2795](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/EmbedMinimalPerf.default.html) | [2851](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.98:1
ProviderMinimalPerf.default | [321](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ProviderMinimalPerf.default.html) | [327](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 0.98:1
TooltipMinimalPerf.default | [1826](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TooltipMinimalPerf.default.html) | [1856](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 0.98:1
ChatDuplicateMessagesPerf.default | [216](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [223](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 0.97:1
MenuButtonMinimalPerf.default | [1482](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1526](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 0.97:1
SegmentMinimalPerf.default | [296](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SegmentMinimalPerf.default.html) | [305](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 0.97:1
ToolbarMinimalPerf.default | [764](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ToolbarMinimalPerf.default.html) | [785](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.97:1
ListCommonPerf.default | [574](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListCommonPerf.default.html) | [599](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ListCommonPerf.default_base.html) | 0.96:1
RadioGroupMinimalPerf.default | [388](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [404](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 0.96:1
ButtonMinimalPerf.default | [121](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ButtonMinimalPerf.default.html) | [127](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 0.95:1
ChatWithPopoverPerf.default | [276](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ChatWithPopoverPerf.default.html) | [291](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.95:1
DividerMinimalPerf.default | [293](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DividerMinimalPerf.default.html) | [310](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/DividerMinimalPerf.default_base.html) | 0.95:1
TreeWith60ListItems.default | [119](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TreeWith60ListItems.default.html) | [129](https://fluentuipr.z22.web.core.windows.net/pull/27506/perf-test-northstar/TreeWith60ListItems.default_base.html) | 0.92:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27506&commitId=230f0f25e60f5196d0484af13ad035e791125ff3&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1929 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|2| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

**Review by miroslavstastny — APPROVED** [review]

File: change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27506&commitId=56e5912f6747bdb0ad29a2592fc103e390b45df6&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1930 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.width constrained.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>

**Pull request merged**